Source/Include/swigwarn.h

```diff
@@ -255,6 +255,11 @@
 
 /* please leave 740-759 free for Python */
 
+#define WARN_C_TYPEMAP_CTYPE_UNDEF            760
+#define WARN_C_UNSUPPORTTED                   779
```

Comment:
Normally we allocate from the first reserved value up, rather than from both ends of the reserved range in.

Reply:
I think the intention was to clearly reserve the entire range, is this really a problem?

Reply:
I guess the real problem here is that each such tiny little arbitrary difference between target languages adds cognitive load and in combination they make it harder to get your head round SWIG in its entirety, and we really need more people who can deal with stuff across SWIG rather than specific to one target language (we really need more of the latter too, but there's a particular shortage of active people in the first category).  Consistency is useful tool for taming complexity.

So if all the existing target languages do something one way, I think there ought to be a really good reason for a new language doing it differently, even if there's logic behind both approaches.

Reply:
I don't want to relaunch all the CI builds just for this, but if I make any other fixes later or if it's really the only thing which prevents this PR from being merged, I'll change the value of `WARN_C_UNSUPPORTTED` to 761.

Reply:
Use [skip ci] or variations in commit message to avoid rerunning CI, see https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs.